src/components/reviews/Review.js

```diff
@@ -0,0 +1,130 @@
+import React from "react";
+import styled from "styled-components/macro";
+import Rating from "@material-ui/lab/Rating";
+
+function ShowReviews({ review, rating }) {
+  return (
+    <WrapperReview>
+      <ReviewStyled>
+        <ImgStyled src="https://source.unsplash.com/random" alt="face" />
```

Comment:
no alt needed, if it doesn't contain further information for visually impaired. or is it for code reviewers to know what kind of picture it should be?